content/zh-cn/docs/concepts/storage/persistent-volumes.md

```diff
@@ -548,7 +548,7 @@ the following types of volumes:
 * {{< glossary_tooltip text="csi" term_id="csi" >}}
 * flexVolume (deprecated)
 * gcePersistentDisk
-* glusterfs
+* glusterfs (deprecated)
```

Comment:
Line 547, 549, and 551 can be localized.

Reply:
Hey @gaogao101, would you like to fix this?

Reply:
@gaogao101 
```suggestion
* glusterfs (已弃用)
```

Reply:
@howieyuen Thank you for your help.